learn/core_concepts/indexes.md

```diff
@@ -130,3 +131,11 @@ Typo tolerance is a built-in feature that helps you find relevant results even w
 You can update the typo tolerance settings using the [update settings endpoint](/reference/api/settings.md#update-settings) or the [update typo tolerance endpoint](/reference/api/typo_tolerance.md#update-typo-tolerance).
 
 [Learn more about typo tolerance](/learn/configuration/typo_tolerance.md)
+
+### Pagination
+
+To protect your database from malicious scraping, Meilisearch returns upto 1000 results for a search query. You can update this using the [update settings endpoint](/reference/api/settings.md#update-settings) or the [update pagination settings endpoint](/reference/api/pagination.md#update-pagination-settings).
+
+We recommend using the [`limit`](/reference/api/search.md#limit) and [`offset`](/reference/api/search.md#offset) search parameters to implement previous and next buttons for a pagination UI.
```

Comment:
I'm not sure if we need this line here, but this section looked too short compared to the rest

Reply:
I think we can skip it. I feel it doesn't offer enough information.

Perhaps we could add a "You can read more about paginating with Meilisearch on our dedicated guide" linking to the page that's on the other pagination PR?